pkg/apis/kops/componentconfig.go

```diff
@@ -234,6 +228,13 @@ type KubeletConfigSpec struct {
 	// ExperimentalAllocatableIgnoreEviction enables ignoring Hard Eviction Thresholds while calculating Node Allocatable
 	ExperimentalAllocatableIgnoreEviction *bool `json:"experimentalAllocatableIgnoreEviction,omitempty" flag:"experimental-allocatable-ignore-eviction"`
 
+	// imageMinimumGCAge is the minimum age for an unused image before it is garbage collected. Default: "2m"
+	ImageMinimumGCAge *string `json:"imageMinimumGCAge,omitempty"`
+	// imageMaximumGCAge is the maximum age an image can be unused before it is garbage collected.
+	// The default of this field is "0s", which disables this field--meaning images won't be garbage
+	// collected based on being unused for too long. Default: "0s" (disabled)
+	ImageMaximumGCAge *string `json:"imageMaximumGCAge,omitempty"`
```

Comment:
Should be backwards compatible.
```suggestion
	ImageMaximumGCAge *Should be backwards compatible. `json:"imageMaximumGCAge,omitempty"`
```